docs/raiden-api-1/api-tutorial/2-open-a-channel.inc.rst

```diff
@@ -88,6 +88,7 @@ the following response object:
    {
        "token_network_address": "0x3C158a20b47d9613DDb9409099Be186fC272421a",
        "channel_identifier": "99",
+       "network_state": "unknown",
```

Comment:
what is this? 🤔

Reply:
Ah, found it below. Maybe not the best naming, but not in scope for this PR:

Reply:
That's the presence information for the participant. This was added at some point and never documented.
Since the presence is now passed through to the transport via the metadata and no state is kept on it, it is not accessible "globally" anymore. Still, to guarantee API compatibility, the `unknown` dummy value is now always returned. 
The docs have been changed to reflect that.